NEWS.md

```diff
@@ -1,8 +1,10 @@
 # poissonreg (development version)
 
+* Predictions via `multi_predict()` for glmnet models now correctly are the mean counts instead of the linear predictor (#89).
```

Comment:
```suggestion
* Predictions via `multi_predict()` for glmnet models now correctly default to mean counts instead of the linear predictor (#89).
```

Maybe? Not sure if this is actually the case.